docker-compose.yml

```diff
@@ -32,6 +33,36 @@ services:
       - CELERY_LOGLEVEL=DEBUG
       - CELERY_WORKER_CONCURRENCY=${CELERY_WORKER_CONCURRENCY:-4}
       - SUBPATH=${SUBPATH:-/}
+      # setup_configuration env vars
+      - OPENNOTIFICATIES_DOMAIN=web:8000
+      - OPENNOTIFICATIES_ORGANIZATION=ON
+      - AUTORISATIES_API_ROOT=https://open-zaak.example.nl/autorisaties/api/v1/
+      - NOTIF_OPENZAAK_CLIENT_ID=notif-client-id
+      - NOTIF_OPENZAAK_SECRET=notif-secret
+      - OPENZAAK_NOTIF_CLIENT_ID=oz-client-id
+      - OPENZAAK_NOTIF_SECRET=oz-secret
+    ports:
+      - 8000:8000
+    depends_on:
+      db:
+        condition: service_started
+      redis:
+        condition: service_started
```

Comment:
In the docker-compose for OZ, there is only a `depends_on` for `web-init`: https://github.com/open-zaak/open-zaak/pull/1572/files#diff-e45e45baeda1c1e73482975a664062aa56f20c03dd9d64a827aba57775bed0d3R59-R61

I assume the fact that `web-init` itself depends on `db` and `redis` makes this work without needing these here explicitly?

Reply:
You are right, I've changed it